index.html

```diff
@@ -0,0 +1,253 @@
+                <p>The best price/quantity plan *</p>
+              </div>
+              <div class="product-price">
+                <h4>$249</h4>
+                <p>Per Month</p>
+              </div>
+
+              <div class="product-description">
+                <div class="product-sub-description">
+                  <h4>Subtitle Goes Here</h4>
+                  <p>Subtitle second inline goes here</p>
+                </div>
+                <ul>
+                  <li>Drop down ramen hustle</li>
+                  <li>Coworrking viral landing</li>
+                  <li>Crush revenue traction</li>
+                  <li>User base minimum viable</li>
+                  <li>Lorem ipsum dolor</li>
+                </ul>
+
+                <button>
+                  <i class="fa-solid fa-caret-right"></i>
+                  ORDER
+                </button>
+              </div>
+            </div>
+            <div class="products-items">
+              <div class="product-title">INSANE</div>
+              <div class="product-price">
+                <h4>$1399</h4>
+                <p>Per Month</p>
+              </div>
+
+              <div class="product-description">
+                <div class="product-sub-description"></div>
+                <ul>
+                  <li>Drop down ramen hustle</li>
+                  <li>Coworrking viral landing</li>
+                  <li>Crush revenue traction</li>
+                  <li>User base minimum viable</li>
+                  <li>Lorem ipsum dolor</li>
+                </ul>
+
+                <button>
+                  <i class="fa-solid fa-caret-right"></i>
+                  ORDER
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- footer -->
+      <div class="footer">
+        <div class="footer-container">
+          <p class="footer-copyright">2021 © <span>CodexThemes</span></p>
+          <ul class="footer-bars">
+            <li><a href="#">Support</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Disclaimer</a></li>
+            <li><a href="#">Add more</a></li>
+          </ul>
+          <ul class="footer-social">
+            <li>
+              <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-instagram"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-pinterest-p"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-youtube"></i></a>
+            </li>
+          </ul>
+          <a href="#" class="go-top"><i class="fa-solid fa-angle-up"></i></a>
```

Comment:
button này không thuộc footer, em đưa button này ra ngoài, sau đó xử lý ẩn, hiện, cố định button scroll to top khi người dùng scroll đến nội dung trang web lớn hơn 100px